pravega/pravega
# Issue 5729: Include LICENSE and NOTICE files into javadoc distribution

**Change log description**  
Modifies javadoc tasks by addinto license files into a set of files to be packaged as a distribution bundle 

**Purpose of the change**  
Fixes #5729 

**What the code does**  
`javadocDistZip` and `javadocDistTar` gradle tasks are modified to include license files into a resulting distribution bundle

**How to verify it**  
Verify that upon unpacking the javadoc distribution LICENSE and NOTICE files are present
